frontend/common/stores/organisation-store.js

```diff
@@ -329,7 +329,11 @@ store.dispatcherIndex = Dispatcher.register(store, (payload) => {
 
   switch (action.actionType) {
     case Actions.GET_ORGANISATION:
-      controller.getOrganisation(action.id || store.id, action.force)
+      controller.getOrganisation(
+        action.id,
+        action.force || store.id,
+        action.force,
```

Comment:
This is quite confusing, what are each of the arguments here?

Reply:
This change was reverted.